lms/envs/aws.py

```diff
@@ -605,3 +605,5 @@
 PROFILE_IMAGE_SECRET_KEY = AUTH_TOKENS.get('PROFILE_IMAGE_SECRET_KEY', PROFILE_IMAGE_SECRET_KEY)
 PROFILE_IMAGE_MAX_BYTES = ENV_TOKENS.get('PROFILE_IMAGE_MAX_BYTES', PROFILE_IMAGE_MAX_BYTES)
 PROFILE_IMAGE_MIN_BYTES = ENV_TOKENS.get('PROFILE_IMAGE_MIN_BYTES', PROFILE_IMAGE_MIN_BYTES)
+if FEATURES['IS_EDX_DOMAIN']:
```

Comment:
Right, since this is in aws.py it's only executed once per lifecycle and it's global. I think this code is right, so you shouldn't change it here. I'll see if I can find the place where we'll need to override it, but it is not in this change set. Give me 5 minutes.